WcaOnRails/db/migrate/20220627195217_add_allow_registration_without_qualification_to_competitions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddAllowRegistrationWithoutQualificationToCompetitions < ActiveRecord::Migration[5.2]
```

Comment:
Your `bin/rails` runner seems to still generate migrations on version 5.2.
When did you run `bundle install` for the last time? 😅